calico/datamodel_v1.py

```diff
@@ -81,6 +87,18 @@ def dir_for_per_host_config(hostname):
     return dir_for_host(hostname) + "/config"
 
 
+def dir_for_status(hostname):
+    return STATUS_DIR + "/%s" % hostname
+
+
+def key_for_status(hostname):
+    return dir_for_status(hostname) + "/last_reported_status"
+
+
+def key_for_uptime(hostname):
+    return dir_for_status(hostname) + "/uptime"
```

Comment:
Since we'll have to live with this API for a long time, I'm wondering if it should have some mention of felix in it since "host/uptime" looks like the uptime of the host, rather than felix.   Maybe just change this to "felix_uptime"?

Also, the string "(felix_)uptime" appears all over the place, it should be in a constant in this file.